pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -36,7 +36,7 @@ import (
 // Client is a KUDO Client providing access to a kudo clientset and kubernetes clientsets
 type Client struct {
 	kudoClientset versioned.Interface
-	kubeClientset kubernetes.Interface
+	kubernetes.Interface
```

Comment:
I'm not sure about this change. At the moment it's rather easy to see that we have two distinct clients here, if we apply this change the `kudoClientset` will mostly be hidden by funcs from the `kubernetes.Interface`. And if we add more clients at some point it gets even worse.

Reply:
I agree it was a hasty decision of mine, I just wanted to expose the required Kube methods.
Previously I was just manually created two separate clients, for example:
https://github.com/kudobuilder/kudo/blob/27fdeb2cd25f37c4be8b64fb98dd95a7534644f3/pkg/kudoctl/cmd/diagnostics/resource_funcs.go#L33-L37
Now we have this unified client which would be convenient to use. Currently in master the embedded Kube client is used only for Namespace creation.
What would you suggest? Or is this client not expected to expose Kube funcs and I should switch back to two clients?

Reply:
I actually didn't mind this change. KUDO types are distinct enough from k8s ones and I see no reason to type `client.kubeClientset.*` every time. But this is a soft opinion.

Reply:
I'm not too fixed on this as well, I just think its clearer when we have the `client.kubeClientset.*`, `client.kudoClientset.*, `client.discovery.*`, `client.dynamic.*`, `client.extensions.*` etc...

Reply:
Perhaps, a consistent approach would be to expose Kube resources via the dedicated functions, like it's done for _GetInstance_. This is compliant with Law of Demeter and as a possible benefit something can be wrapped into the methods (like, setting GVKs).
However, inlining is just must lighter.
From the other hand, maybe I just misunderstand the purpose of this client, and it just _should not_ be used to fetch Kube resources?
It seems, some small discussion is needed.